opmd_viewer/addons/pic/lpa_diagnostics.py

```diff
@@ -160,12 +160,12 @@ def get_sigma_gamma_slice(self, dz, t=None, iteration=None, species=None,
         if plot:
             check_matplotlib()
             iteration = self.iterations[ self._current_i ]
-            time_fs = 1.e15 * self.t[ self._current_i ]
+            time = self.t[ self._current_i ]
```

Comment:
maybe it's still useful to see the time is already in SI here, e.g. varname `time_s` ?